tests/test_rhiza/test_book.py

```
@@ -0,0 +1,66 @@
+
+MAKE = shutil.which("make") or "/usr/bin/make"
+
+
+def test_no_book_folder(git_repo):
+    """Test that make targets fail gracefully when book folder is missing."""
+    shutil.rmtree(git_repo / "book")
+    assert not (git_repo / "book").exists()
+
+    for target in ["book", "docs", "marimushka"]:
+        # test resilience
+        result = subprocess.run([MAKE, target], cwd=git_repo, capture_output=True, text=True)
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)